admin/sales_analytics/views.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+from admin.sales_analytics import keen
+from admin.sales_analytics.metrics import user_count, multi_product_metrics_yearly, multi_product_metrics_monthly, repeat_action_user_monthly
+
+
+def dashboard(request):
```

Comment:
Even though this is behind a VPN we should consider putting security on these. It also might be nice to take a CBV approach in order to make this simpler to do and test. Please look at other admin modules in order to get an idea.